### ISSUE TITLE
Auto rotate a bit too aggressive on pixel 4a android 

### ISSUE THREAD

**Issue by skermiebroTech** [futo-org/grayjay-android#1022]
It feels like any time I even breathe on my phone the screen rotates if I don't have rotation lock on. Maybe a slider to adjust sensitivity? Ty for making an awesome app :)

**Comment by teaalltr** [comment]
The setting is already there, check the app settings

**Comment by SeveringHams** [comment]
its also an issue on TCL 10 Pro. It seems to just take the raw rotation input and directly control the screen rotation with it, instead of averaging it over a second and a half or something.

**Comment by lucina-da-lucario** [comment]
Yeah, I have this problem as well (Samsung Galaxy J3 V). Might be a thing across the board.

**Comment by lucina-da-lucario** [comment]
> The setting is already there, check the app settings

I just gave it a whirl. This *does* help, but it might also help if there was a timer so that after, say, a second or two at most, the screen can rotate again. Right now, it can rotate immediately after going to a new orientation (which plays into a bug where the UI duplicates and overlaps), which can be a little annoying depending on what you have it set to.

**Comment by skermiebroTech** [comment]
> > The setting is already there, check the app settings
> 
> I just gave it a whirl. This *does* help, but it might also help if there was a timer so that after, say, a second or two at most, the screen can rotate again. Right now, it can rotate immediately after going to a new orientation (which plays into a bug where the UI duplicates and overlaps), which can be a little annoying depending on what you have it set to.

Have you replaced the UI duplication bug? ![Screenshot_20240526-182450.png](https://github.com/futo-org/grayjay-android/assets/67085932/49fa6cc3-4e81-49e4-a940-24531a1ca112)

**Comment by lucina-da-lucario** [comment]
> Have you replaced the UI duplication bug? 

Not sure what you mean by replaced. Can you clarify?

**Comment by skermiebroTech** [comment]
> > Have you replaced the UI duplication bug? 
> 
> Not sure what you mean by replaced. Can you clarify?

Sorry autocorrect strikes again I meant reported

**Comment by lucina-da-lucario** [comment]
> > > Have you replaced the UI duplication bug?
> > 
> > 
> > Not sure what you mean by replaced. Can you clarify?
> 
> Sorry autocorrect strikes again I meant reported

Ah. Yes, I have, actually. Someone else reported the same thing after I did, in fact. https://github.com/futo-org/grayjay-android/issues/989

**Comment by dmarszk** [comment]
Same issue here. Even with the biggest rotation margin selected. This can be solved in a number of way, but the simplest might be debouncing like - whenever a rotation occurs, store its timestamp and become insensitive for further rotations for ~700ms.
